### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by mattsym** [Mwyann/psakey#5]
I think are u here :) Do u know it still work ? I wnat use pi 4 4gb what u think about it i completely beginner heh

**Comment by Mwyann** [comment]
Yes it still works, and although the RPi 4's USB C port can be used as a data port, I didn't test it myself. I'm pretty sure the script won't work OOTB though, especially for the kernel's module part. Also, I think such device is kinda overkill for the usage, that's why I'd suggest using a RPi Zero instead, as it uses much less power but is powerful enough to run the needed software (unless you had some crazy usage in mind, which I would love to hear about).
